src/olympia/addons/indexers.py

```diff
@@ -165,7 +163,14 @@ def get_mapping(cls):
                         'properties': {
                             'id': {'type': 'long', 'index': False},
                             'modified': {'type': 'date', 'index': False},
-                            'sizes': {'type': 'keyword', 'index': False},
+                            'sizes': {
+                                'type': 'object',
+                                'properties': {
+                                    'thumbnail': {'type': 'short',
+                                                  'index': False},
```

Comment:
Let's be explicit and format this like

```python
'thumbnail': {
    'type': 'short',
    'index': False
},
```

imho we do that elsewhere too in the schema

Reply:
I can't see where we do, exactly.  `name` above is written in same style as `thumbnail`, and some of the version mappings are `'xxx': {\n'type': 'keyword', 'index': False}` style.